### PR TITLE
Test if samples are running

### PR DESCRIPTION
**Pull request by rakshith91**
Fixes https://github.com/Azure/azure-sdk-for-python/issues/11152

Live CI: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=414595&view=results

### CODE REVIEW

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/test_samples.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    )
+    stdout, stderr = proc.communicate()
+
+    return proc.returncode, stdout, stderr
+
+def _test_file(file_name, account, key, root_dir='./samples'):
+    os.environ['AZURE_FORM_RECOGNIZER_ENDPOINT'] = account
+    os.environ['AZURE_FORM_RECOGNIZER_KEY'] = key
+    code, _, err = run([sys.executable, root_dir + '/' + file_name])
+    print(_)
```
Comment (rakshith91) [inline sdk/formrecognizer/azure-ai-formrecognizer/tests/test_samples.py:45]: will remove the print statemtns

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/test_samples.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    return proc.returncode, stdout, stderr
+
+def _test_file(file_name, account, key, root_dir='./samples'):
+    os.environ['AZURE_FORM_RECOGNIZER_ENDPOINT'] = account
+    os.environ['AZURE_FORM_RECOGNIZER_KEY'] = key
+    my_env = dict(os.environ)
+    if sys.version_info < (3, 5):
+        my_env = {key: str(val) for key, val in my_env.items()}
+    code, out, err = run([sys.executable, root_dir + '/' + file_name], my_env=my_env)
+    print(out)
```
Comment (rakshith91) [inline sdk/formrecognizer/azure-ai-formrecognizer/tests/test_samples.py:49]:
```suggestion
    code, _, err = run([sys.executable, root_dir + '/' + file_name], my_env=my_env)
```

**Comment by kristapratico** [comment]
Can you check the live build again? It's showing all green, but also showing sample tests that failed

File: sdk/formrecognizer/azure-ai-formrecognizer/tests/test_samples_async.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    os.environ['AZURE_FORM_RECOGNIZER_ENDPOINT'] = account
+    os.environ['AZURE_FORM_RECOGNIZER_KEY'] = key
+    code, _, err = run([sys.executable, root_dir + '/' + file_name], my_env=dict(os.environ))
+    assert code == 0
+    assert err is None
+
+
+class TestSamplesAsync(FormRecognizerTest):
+    # Async sample tests
+    @pytest.mark.live_test_only
```
Comment (iscai-msft) [inline sdk/formrecognizer/azure-ai-formrecognizer/tests/test_samples_async.py:51]: Why are we marking samples as live test only?

**Comment by iscai-msft** [comment]
Also I don't know if this will change the structure too much, but would it be possible to have the sample tests run in their own step. Ideally the samples could also run in playback so when we make changes, we could more easily see immediately if we are breaking the samples

**Comment by rakshith91** [comment]
We will be able to run samples on recordings when we have it’s own step in CI sometime in the future.  For now, we have to use some of the env variables used in love tests. That’s why they are only live.

**Comment by rakshith91** [comment]
@kristapratico that was because I had to re run the job for some reason and then they passed.

**Pull request merged**